packages/plugin-survey/src/index.ts

```diff
@@ -1,48 +1,104 @@
 import { JsPsych, JsPsychPlugin, ParameterType, TrialType } from "jspsych";
-import { StylesManager, Survey } from "survey-knockout";
+import {
+  QuestionCheckbox,
+  QuestionComment,
+  QuestionDropdown,
+  QuestionMatrix,
+  QuestionRadiogroup,
+  QuestionRanking,
+  QuestionSelectBase,
+  StylesManager,
+  Survey,
+} from "survey-knockout";
+//import { require } from "yargs";
```

Comment:
whoops :see_no_evil: At least I'm not the only one to be sometimes overcome by VSCode's quick-acting autocompletion :facepalm:

Reply:
Haha, I thought this was something you added and was confused 😆